scripts/run_bm.sh

```diff
@@ -32,6 +32,7 @@ function build_narayana {
         git clone https://github.com/${NARAYANA_REPO}/narayana.git -b ${NARAYANA_BRANCH} narayana-tmp
         [ $? = 0 ] || fatal "git clone https://github.com/${NARAYANA_REPO}/narayana.git failed"
       else
+        NARAYANA_BRANCH="${NARAYANA_BRANCH:-${GIT_BRANCH}}"
```

Comment:
When narayana-tmp from a previous script run is used, the NARAYANA_BRANCH may not have been exported so set again

Reply:
ok